jolly-jellyfish/src/page_maker/templates/page_maker/example_page.html

```diff
@@ -0,0 +1,15 @@
+{% extends "page_maker/base.html" %}
+
+{% block content %}
+<h1><span>Example Page title</span></h1>
```

Comment:
This is required to be `<h1><span><span>Example Page title</span></span></h1>` by tht hotdog theme.

Reply:
```suggestion
<h1><span><span>Example Page title</span></span></h1>
```